tests/test_infosets.py

```diff
@@ -52,3 +60,47 @@ def test_infoset_add_action_error():
     game = games.read_from_file("basic_extensive_game.efg")
     with pytest.raises(gbt.MismatchError):
         game.add_action(game.players[0].infosets[0], game.players[1].infosets[0].actions[0])
+
+def test_append_move_single_infoset_ist_of_nodes():
+    """Test that appending a list of nodes creates a single infoset."""
+    game = _create_game()
+    nodes = [game.root.children[1],
+             game.root.children[0].children[0],
+             game.root.children[0].children[1]]
+    game.append_move(nodes, "Player 3", ["B", "F"])
+
+    assert len(game.players[2].infosets) == 1
```

Comment:
It would be more transparent to be consistent and index player by label in the assertion.,